ext/ldap/ldap.c

```diff
@@ -3420,7 +3420,7 @@ PHP_FUNCTION(ldap_parse_exop)
 	int rc, myargcount = ZEND_NUM_ARGS();
 
 	if (zend_parse_parameters(myargcount, "rr|zz", &link, &result, &retdata, &retoid) != SUCCESS) {
-		WRONG_PARAM_COUNT;
```

Comment:
Am I right that it is not needed anymore?

Reply:
It throws an exception.

I think we can open a PR removing that as now `zend_parse_parameters` thrown one making impossible to reach 

https://github.com/php/php-src/blob/e63a44dd03aae264561fff77565f4a5f9be035bd/Zend/zend_API.c#L88-L95

Reply:
Yep, I saw that `zend_argument_count_error()` throws an exception, but couldn't imagine what would happen afterwards. :)

Reply:
Your change is right ... the current code is going to throw two chained exceptions, which it shouldn't...